### ISSUE TITLE
Client: Merge Work Procedure

### ISSUE THREAD

**Issue by holgerd77** [ethereumjs/ethereumjs-monorepo#1507]
As a work precedure on Merge-related work I suggest that we add a new branch `merge` where we only activate the `client` and eventually the `blockchain` tests as being mandatory (+ maybe one `client-cli test?), so that it's possible to do some quicker work iterations while maintaining a minimal level of integrity.

I've got some tendency to continue to do non-Merge related work (but which is done in a merge related context, e.g. an eventual websocket support addition) still towards `master` and then do in-between rebases of the `merge` branch.

We can than use this branch to deploy on our server (by using the dev install instructions from the [client README](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/client#install) and then add as a service.

We likely want to add this as an additional service to the existing `ethereumjs.service` client release deployment, e.g. `ethereumjs-merge.service` and also use a separate datadir on the command line.

Then debugging work on `ethereumjs.service` can also continue in parallel. Not sure, is it generally possible to run two instances of our client on the same machine? 

We should also test if the RPC port on the server can be reached from the outside (please do not expose the IP here though). Maybe someone can already test this on the `ethereumjs.service` client by adding the `--rpc` flag, restart and trying to connect.

**Comment by holgerd77** [comment]
That said: if you want to implement something in doubt don't wait for a `merge` branch, I guess we can merge most merge-related work directly into `master` relatively without problems as well (as we have already done).
